docs/bcfusion/validators/creation.md

```diff
@@ -68,13 +71,17 @@ geth_macos bls account generate-proof --chain-id ${BSC_CHAIN_ID} ${OPEATOR_ADDRE
 An example proof
 is `0xaf762123d031984f5a7ae5d46b98208ca31293919570f51ae2f0a03069c5e8d6d47b775faba94d88dbbe591c51c537d718a743b9069e63b698ba1ae15d9f6bf7018684b0a860a46c812716117a59c364e841596c3f0a484ae40a1178130b76a5`.
 
-- `BSC_CHAIN_ID`: 56 for BSC mainnet, and 97 for BSC testnet.
+- `BSC_CHAIN_ID`: `56` for BSC mainnet, and `97` for BSC testnet.
 - `OPEATOR_ADDRESS`: The address of your account, which will be reconginzed as the operator of the new validator.
 - `VOTE_ADDRESS`: The vote address created in the last step.
 
 ### Create indentity
 
-Please down load BC client binary here: https://github.com/bnb-chain/node/releases/tag/v0.10.18
+Identity is used for assocaiting the new validator to the old validator created on the Beacon Chain, to facilitate
+delegators moving their stakes to the same validator operator when [migrations](../users/stake-migration.md).
+If you are creating a new validator without any need for such association, you can leave it empty.
```

Comment:
`If you are creating a new validator without any need for such association` ==> `If you never create a validator in this [page](https://www.bnbchain.org/en/staking)`

Reply:
updated